src/app/collective-rewards/rewards/MyRewards.tsx

```diff
@@ -22,15 +22,35 @@ import { RequiredBuilder } from '@/app/collective-rewards/types'
 import { Switch, SwitchThumb } from '@/components/Switch'
 import { Typography } from '@/components/Typography'
 import { CycleContextProvider } from '@/app/collective-rewards/metrics'
+import { useAccount } from 'wagmi'
+import { Button } from '@/components/Button'
+import Image from 'next/image'
 
-const SubText = () => {
-  return (
-    <>
-      Track and claim the rewards you earn from Collective Rewards. For more information check the{' '}
-      <CRWhitepaperLink />.
-    </>
-  )
-}
+const SubText = () => (
+  <>
+    Track and claim the rewards you earn from Collective Rewards. For more information check the{' '}
+    <CRWhitepaperLink />.
+  </>
+)
+
+const NotBacking = () => (
+  <div className="flex flex-col gap-y-[32px] items-center justify-center">
+    <div className="flex flex-col gap-y-[8px] items-center">
+      <Typography tagVariant="label" className="text-2xl font-normal font-kk-topo leading-[33.60px]">
+        Not backing any builder yet.
+      </Typography>
+      <Typography tagVariant="label" className="text-base font-normal font-rootstock-sans leading-snug">
+        Support builders by allocating your stRIF and start earning rewards.
+      </Typography>
+    </div>
+    <div>
+      <Button variant="white-new" data-testid="BrowBuildersButton">
+        Browse Builders
```

Comment:
@antomor I don't know what is the action from this button and even if it's going to have a tooltip when the wallet is not connected.

Reply:
it should redirect you to the "Collective Rewards" page

Reply:
I've directly applied the required changes 👍🏼